addons/event/models/event_event.py

```diff
@@ -193,6 +193,7 @@ def _default_event_mail_ids(self):
     address_id = fields.Many2one(
         'res.partner', string='Venue', default=lambda self: self.env.company.partner_id.id,
         tracking=True, domain="['|', ('company_id', '=', False), ('company_id', '=', company_id)]")
+    address_search = fields.Many2one('res.partner', search='_search_address_search', readonly=True, store=False)
```

Comment:
Can't we give it a value in a compute (would be address_id) ? Or related='address_id' ? That way it has a value :)